apps/site/lib/site_web/views/trip_plan_view.ex

```diff
@@ -558,7 +558,7 @@ defmodule SiteWeb.TripPlanView do
     HTML.Tag.content_tag(
       :span,
       [
-        "Total may be less with ",
+        "Total, may be less with ",
```

Comment:
Do we actually want the comma in the accordion, but not in the summary above? Maybe a question for Katrina? (I am probably not the person to ask about grammar questions…)

Reply:
Ah, good point. I added the comma here because that's what was in the design but I will double check with Katrina.